packages/devextreme/js/__internal/ui/calendar/m_calendar.ts

```diff
@@ -251,116 +261,134 @@ class Calendar<
 
         this._moveToClosestAvailableDate(date);
       },
-      pageUp(e): void {
+      pageUp(e: KeyboardEvent): void {
```

Comment:
Now, e process correctly without additional definition - it should be DxEvent<KeyboardEvent>. I suppose you can omit type for 'e' or change it to DxEvent<KeyboardEvent>

Reply:
https://github.com/DevExpress/DevExtreme/commit/ec2f610d374bc1eb469a6ab73af78a85080ffa5d